src/classes/CRLP_DefaultConfigBuilder_SVC.cls

```diff
@@ -45,6 +45,15 @@ public class CRLP_DefaultConfigBuilder_SVC {
             WHERE SobjectType = 'Opportunity' AND IsActive = True]);
 
 
+    /*************************************************************************************************************
+     * @description Call each of the methods in this class to build the default Filter Groups and each of the four
+     * Rollup Types (Account Hard Credit/Membership, Contact Hard Credit/Membership, Contact Soft Credit, and GAU)
+     */
+    @future
+    public static void convertLegacyRollupsIntoCustomizableRollupsFuture() {
```

Comment:
This isn't needed anymore as we're updating the custom settings in the callback, right?